src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  max-height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo_link {
+  display: inline-block;
+  margin: 10px 50px;
+  height: 40px;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  margin-right: 20px;
+  list-style-type: none;
+}
+
+.nav_list:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  display: block;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  width: 37px;
```

Don't hardcode the width
```suggestion
  width: 100%;
```